### ISSUE TITLE
pictograms

### ISSUE THREAD

**Issue by JohanKrab** [pippyn/Home-Assistant-Sensor-Afvalbeheer#445]
After installation, i cannot find the pictograms to assign to the entities.
This is my configuration:
afvalbeheer:
  wastecollector: Waardlanden
  resources:
    - restafval
    - gft
    - papier
    - pmd
  postcode: xxxxxx
  streetnumber: xx
  buildinicons: 1
  builtiniconsnew: 1
  dateformat: '%d %b'
  disableicons: 1
  dutch: 1

**Comment by pippyn** [comment]
Remove the `disableicons: 1` option

**Comment by JohanKrab** [comment]
Remove the option didn´t solve the issue

**Comment by pippyn** [comment]
Did you reboot HA? Because without that option the icons should appear

**Comment by JohanKrab** [comment]
Yes several times. I even removed all and install it via HACS but not solving the issue.

**Comment by pippyn** [comment]
Do you have an address for me to test with?

**Comment by JohanKrab** [comment]
Sorry, but i don´t have remote access

**Comment by pippyn** [comment]
I mean a street address in your neighborhood so I can test if there is something wrong with the Waardlanden integration

**Comment by JohanKrab** [comment]
Klaverkamp

**Comment by HooghiemstraJ** [comment]
variable results on icons and on sortibak/restafval

yes: did a hard reset, 
yes: restarted HA
yes:  reload configuration.yaml

Configuration.yaml:

afvalbeheer:
      wastecollector: Omrin
      resources:
        - restafval
        - gft
        - papier
        - pmd
        - sortibak
      postcode: 9283TE
      streetnumber: 12
      upcomingsensor: 1
      dateformat: '%d-%m-%Y'
      dateonly: 0
      builtinconsnew: 1
      dutch: 1

![Container](https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer/assets/67979049/77b83032-cc53-4025-888c-099f51138883)

**Comment by pippyn** [comment]
First of all this issue is about Waardlanden, not Omrin.
About your issue; not all fractions have a built in icons.

**Comment by dennisb1** [comment]
I'm using omrin but i don't get the icons at all.
Also manual searching for them wont help...

yes: did a hard reset,
yes: restarted HA
yes: reload configuration.yaml

**Comment by pippyn** [comment]
What is your config?

**Comment by dennisb1** [comment]
```
afvalbeheer:
  wastecollector: Omrin
  resources:
    - Sortibak
    - Papier
    - GFT
    - Chemisch
    - Grofvuil
  postcode: 893xxx
  streetnumber: xxx
  builtinicons: 1
```

Also tried adding   `builtinicons: 1`

**Comment by pippyn** [comment]
Yes, use the buitlinicons option. But only Papier and GFT are supported.

**Comment by dennisb1** [comment]
This is the result:
![image](https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer/assets/29853030/4692a46d-0cfe-4699-b541-5f0fe4beae0e)

Also as a separate quistion, i'm not good in HA but is it possible to instead of a date say 4 days left or something?

**Comment by pippyn** [comment]
You'll need to reboot HA and wait a couple of minute.

You can use the upcoming sensors for that: https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer?tab=readme-ov-file#upcoming-sensor

**Comment by dennisb1** [comment]
> You'll need to reboot HA and wait a couple of minute.

It's already in this state for 48 hours, i rebooted before that already 3-4 times, integration is already for 14 days in my HA.
Also why only Papier and GFT?  When reading the manual it should support more.
https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer?tab=readme-ov-file#built-in-icons

> You can use the upcoming sensors for that: https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer?tab=readme-ov-file#upcoming-sensor
Ah upcoming.. missed that when reading, thank you for that!

**Comment by wolfpins** [comment]
Why i don't get a icon on opk and on gft and pmd well (Maassluis)?
